tour6-deploying-with-promotions_5bb91da12c7d3a04dd5b5d8f.md

```diff
@@ -1,36 +1,43 @@
-Each project starts with the default pipeline specified in
+Each Semaphore 2.0 project starts with the default pipeline specified in
 `.semaphore/semaphore.yml`. Real world pipelines tend to branch out
-when certain conditions are met. Examples may be deploying to
-production on master builds or deploying to a pre-production environment
-on topic branches.
+when certain conditions are met. Examples may be deploying to production on
+master builds or deploying to a pre-production environment on topic branches.
 
-On Semaphore deployment and delivery is managed with _promotions_, which may
-be automatic or manual and optionally depend on conditions.
+On Semaphore 2.0, deployment and delivery is managed with _promotions_, which
+may be performed automatically or manually and may optionally depend on
+user-defined conditions.
 
 ## Manual deployment
 
 Let's start by adding a manual confirmation to promote to production.
 
-<pre><code class="language-yaml"># .semaphore/semaphore.yml
+<pre><code class="language-yaml">
+# .semaphore/semaphore.yml
+version: v1.0
+name: Promotions and Auto-promotions
 agent:
   machine:
     type: e1-standard-2
     os_image: ubuntu1804
+
 blocks:
-  - name: Test
+  - name: Promotions
     task:
       jobs:
-        - name: 'Everything'
+        - name: Everything
           commands:
-            - echo 'running tests'
+            - echo 'Running tests'
 promotions:
   - name: Production deploy
     pipeline_file: production-deploy.yml
 </code></pre>
 
 Now create a new pipeline file in `.semaphore/production-deploy.yml`:
 
-<pre><code class="language-yaml"># .semaphore/production-deploy.yml
+<pre><code class="language-yaml">
+# .semaphore/production-deploy.yml
+version: v1.0
+name: File .semaphore/production-deploy.yml
```

Comment:
name should be something descriptive like "Deploy to production".

Reply:
Done.